drivers/storage/rackspace/executor/rackspace_executor.go

```diff
@@ -0,0 +1,354 @@
+package executor
+
+// // StorageExecutor is the part of a storage driver that is downloaded at
+// // runtime by the libStorage client.
+// type StorageExecutor interface {
+// 	Driver
+// 	StorageExecutorFunctions
+// }
+
+// // StorageExecutorFunctions is the collection of functions that are required
+// // of a StorageExecutor.
+// type StorageExecutorFunctions interface {
+// 	// InstanceID returns the local system's InstanceID.
+// 	InstanceID(
+// 		ctx Context,
+// 		opts Store) (*InstanceID, error)
+
+// 	// NextDevice returns the next available device.
+// 	NextDevice(
+// 		ctx Context,
+// 		opts Store) (string, error)
+
+// 	// LocalDevices returns a map of the system's local devices.
+// 	LocalDevices(
+// 		ctx Context,
+// 		opts *LocalDevicesOpts) (*LocalDevices, error)
+// }
+
+import (
+	//"bufio"
+	//"fmt"
+	//"io"
+	"io/ioutil"
+	//"os"
+	"os/exec"
+//	"path/filepath"
+	"regexp"
+	"strings"
+	
+	"github.com/akutz/gofig"
+	"github.com/akutz/goof"
+
+	"github.com/emccode/libstorage/api/registry"
+	"github.com/emccode/libstorage/api/types"
+	"github.com/emccode/libstorage/drivers/storage/rackspace"
+
+)
+
+// driver is the storage executor for the VFS storage driver.
+type driver struct {
+	config gofig.Config
+}
+
+// const (
+// 	mountinfoFormat = "%d %d %d:%d %s %s %s %s"
+// )
+
+func init() {
+	registry.RegisterStorageExecutor(rackspace.Name, newDriver)
+}
+
+func newDriver() types.StorageExecutor {
+	return &driver{}
+}
+
+func (d *driver) Init(ctx types.Context, config gofig.Config) error {
+	d.config = config
+	return nil
+}
+
+func (d *driver) Name() string {
+	return rackspace.Name
+}
+
+// InstanceID returns the local instance ID for the test
+func InstanceID(config gofig.Config) (*types.InstanceID, error) {
+	//fmt.Println("calling debug InstanceID")//DEBUG
+	d := newDriver()
+	d.Init(nil, config)
+	return d.InstanceID(nil, nil)
+}
+
+// InstanceID returns the aws instance configuration
+func (d *driver) InstanceID(
+	ctx types.Context,
+	opts types.Store) (*types.InstanceID, error) {
+	//fmt.Println("calling InstanceID")//DEBUG
```

Comment:
Hi @akiraw,

Please go through these files and remove any place where the `fmt.Println` is used to note workflow progression. Instead please just use `ctx.Debug` if you need to do this. No need to comment these out when using the logger as they will be ignored if the log level doesn't match. Thanks!